data/policies/derive.cas

```diff
@@ -0,0 +1,82 @@
+virtual resource foo {
+	fn read(domain source) {
+		allow(source, this, file, read);
+	}
+
+	@associated_call
+	fn some_associated_call(domain source) {
+		allow(source, this, file, link);
+	}
+}
+
+virtual resource bar {
+	fn read(domain source) {
+		allow(source, this, dir, read);
+	}
+
+	fn write(domain source) {
+		allow(source, this, dir, write);
+	}
+
+	@associated_call
+	fn some_associated_call(domain source) {
+		allow(source, this, dir, add_name);
+	}
+}
+
+resource custom_define inherits foo, bar {
+	fn read(domain source) {
+		allow(source, this, lnk_file, read);
+	}
+}
+
+@derive([read], parents=*)
```

Comment:
The `parent=*` named argument is optional right? Can we add a case without this argument?
If it is not optional, why do we need to use a named parameter?

Reply:
Yeah, good call.  I've pushed an update adding this test.